.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```
@@ -0,0 +1,84 @@
+name: Bug Report
+description: File a bug report
+title: ""
+labels: [bug, triage]
```

```suggestion
labels: [bug, needs-triage]
```
Sorry I missed this in the earlier review! 😊